src/developer/web-api/new-tracker.md

```diff
@@ -337,6 +337,45 @@ Tracker notes do not have their dedicated endpoint; they are exchanged as part o
 > One between `uid` or `username` field must be provided. If both are provided, only username is considered.
 
 
+### Program stage working lists { #webapi_working_list_filters } 
+
+The program stage working lists are used in the Capture app to present predetermined working lists that are relevant to a specific program stage in the tracker user interface. To interact with them, you'll need to use the */api/trackedEntityInstanceFilters* resource. These lists can be shared and follow the same sharing pattern as any other metadata object. When using the */api/sharing* the type parameter will be *programStageWorkingLists*.
+
+    /api/40/programStageWorkingLists
+
+
```

Comment:
I know we don't have a markdown style guide 😅 I would stick to one newline before and after sections and only one between paragraphs.